packages/nitro-protocol/src/transactions.ts

```diff
@@ -2,7 +2,7 @@ import {Contract, providers, Signature} from 'ethers';
 
 import {Uint256} from './contract/types';
 import {State} from './contract/state';
-import * as forceMoveTrans from './contract/transaction-creators/force-move';
+import * as challengeTrans from './contract/transaction-creators/force-move';
```

Comment:
The changes to this file don't look quite right, since forceMove here refers to the contract name (which we are keeping). Will fix.